prismic-model/js/pages.js

```diff
@@ -24,6 +24,14 @@ const Page = {
     seasons: list('Seasons', {
       season: link('Season', 'document', ['seasons'], 'Select a Season'),
     }),
+    landingPages: list('Parent landing page', {
+      'landing-page': link(
```

Comment:
```suggestion
      landingPage: link(
```

We should use `camelCase` keys.

Do we gain anything from labelling "parent" or hyphenating sometimes? Could we keep the convention as the above - `L/landing page`.